upgrade/helpers/tasks.py

```diff
@@ -748,10 +748,12 @@ def upgrade_using_foreman_maintain():
     TO_VERSION
         To which Satellite version to upgrade.
         e.g '6.2','6.3'
+    :param bool sat_host: if sat_host is True then upgrade will be
+     satellite otherwise capsule.
     """
     env.disable_known_hosts = True
     # setup hammer config
-    if os.environ.get('FROM_VERSION') != "6.3":
+    if sat_host:
```

Comment:
You removed the check of 6.3 which should be added here and after `and`.

Reply:
> You removed the check of 6.3 which should be added here and after `and`.

It is not required here, We have maintained branching kind of stuff for each release and we pick the branch as per their base version. The setting is only applicable for the satellite that's the reason I put this check.